main/webapp/modules/core/styles/common.css

```diff
@@ -64,6 +64,7 @@ table {
 
 input, select {
   vertical-align: middle;
+  margin-bottom: 8px;
```

Comment:
I would rather not indiscriminately add a margin to all `input` and `select` elements in the application as this can have consequences in a lot of places! Please add a narrower selector, ensuring you only style things within the dialog you are working on.

Reply:
Done.